services/wallet/router/fees/fees.go

```diff
@@ -50,23 +56,28 @@ type SuggestedFeesGwei struct {
 	MaxFeePerGasLow      *big.Float `json:"maxFeePerGasLow"`
 	MaxFeePerGasMedium   *big.Float `json:"maxFeePerGasMedium"`
 	MaxFeePerGasHigh     *big.Float `json:"maxFeePerGasHigh"`
```

Comment:
shall we adjust our various fees level?

Reply:
How do you mean this? To have a new formula for each?

This is what we have now:
```golang
{
	Low:    (*hexutil.Big)(new(big.Int).Add(baseFee, maxPriorityFeePerGas)),
	Medium: (*hexutil.Big)(new(big.Int).Add(new(big.Int).Mul(baseFee, big.NewInt(2)), maxPriorityFeePerGas)),
	High:   (*hexutil.Big)(new(big.Int).Add(new(big.Int).Mul(baseFee, big.NewInt(3)), maxPriorityFeePerGas)),
}
```
Let me know if you want to change the equation and to what?

Reply:
I would suggest: high to be 2x base fee and medium to be 1.2 base fee

Reply:
@alaibe I've created a new PR for that, please check it here https://github.com/status-im/status-go/pull/6156